rioting-daisies/BangazonWorkforce
# Index and Create for TrainingPrograms

# Description

This pull request contains the code for the training programs controller and the index view and create view. This provides functionality for the user to view all the training programs in the index view. It also provides functionality to create a new training program.

Fixed bug in the layout.cshtml for the department index view.

Fixes Issue # [(9)](https://github.com/rioting-daisies/BangazonWorkforce/issues/9)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout blue-trainingPrograms`
3. Review the code in the `TrainingProgramsController.cs` and the `Views/TrainingPrograms/Index.cshtml` and the `Views/TrainingPrograms/Create.cshtml`
4. Run the application and click around on the nav links to confirm they all work properly
5. Navigate to training programs and confirm the index view works
6. Click create new training program at the top, and create a new training program


# Checklist:
